### ISSUE TITLE
(fchollet): support masking; in practice `_keras_mask` is never 

### ISSUE THREAD

**Issue by piascikj** [imdone/tensorflow#3790]
set in this context currently.

----
*opened via [imdone.io](https://imdone.io) from a code comment on [4774889094d3f1787a38cfbeb0670cb4fb6e24ff](https://github.com/imdone/tensorflow/commit/4774889094d3f1787a38cfbeb0670cb4fb6e24ff) by Francois Chollet fchollet@google.com*

----

https://github.com/imdone/tensorflow/blob/8cb6e535f6ea14b380aa86d425169adad682cb8c/tensorflow/python/keras/_impl/keras/engine/training_eager.py#L146-L152
